velox/exec/Window.cpp

```diff
@@ -32,12 +32,16 @@ Window::Window(
           "Window"),
       numInputColumns_(windowNode->sources()[0]->outputType()->size()),
       windowBuild_(std::make_unique<SortWindowBuild>(windowNode, pool())),
+      windowNode_(windowNode),
       currentPartition_(nullptr),
-      stringAllocator_(pool()) {
-  auto inputType = windowNode->sources()[0]->outputType();
-  createWindowFunctions(windowNode, inputType, driverCtx->queryConfig());
+      stringAllocator_(pool()) {}
 
+void Window::initialize() {
+  Operator::initialize();
```

Comment:
It may not be obvious that this call is necessary? Let's update comments for Operator::initialize() to make that clear.

Reply:
@mbasmanova thanks and updated!